js/githubresume.js

```diff
@@ -300,7 +300,9 @@ var run = function() {
                     name: lang,
                     popularity: languages[lang],
                     toString: function() {
-                        return '<a href="https://github.com/languages/' + this.name + '">' + this.name + '</a>';
+                        return '<a href="https://github.com/search?q=user%3A'
+                            + username + '&type=Repositories&ref=advsearch&l='
+                            + this.name + '">'+ this.name + '</a>';
                     }
```

Comment:
I think `'<a href="https://github.com/search?q=user:' + username + '&l=' + this.name  + '">' + this.name + '</a>' ` is ok.
`type=Repositories&ref=advsearch ` is not necessary. 

Please rebase it. Thank you.